client/mobile/components/student/responses/multiChoice.js

```diff
@@ -1 +1,86 @@
-multiChoice.js
\ No newline at end of file
+var React = require('react-native');
+
+var {
+  View,
+  Text,
+  StyleSheet,
+} = React;
+
+class MultiChoice extends React.Component {
+  constructor (props){
+    super(props)
+  }
+  
+  previousPage() {
+    this.props.navigator.pop();
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.halfHeight1}>
+          <Text onPress={this.previousPage.bind(this)}>back</Text>
+        </View>
+        <View style={styles.halfHeight}>
+          <Text>MultiChoice</Text>
+        </View>
+        <View style={styles.quarterHeight}>
```

Comment:
Could we nest touchableOpacity components to make these buttons provide feedback to touch?